### PR TITLE
Added info-icon on alerts page

### LINKED ISSUE #260: [UI] Add info icon for on alerts page for add contact points for channel id and slack token
# Description
Add info icon to on alerts page for  add contact points for channel id and slack token.
# Benefits
This will easy for user who is adding contact points. They will know what channel id and slack token looks like.

image

### PR DESCRIPTION
**Pull request by PratikshaPatil77**
# Description
Added info icon on alerts page for add contact points for channel id and slack token.

Fixes #260 

Screenshot 2024-01-23 at 4 21 16 PM

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ x ] I have self-reviewed this PR.
- [ x ] I have removed all print-debugging and commented-out code that should not be merged.
- [ x ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Comment by PratikshaPatil77** [comment]
@nbhavana @sonamgupta21 @sunitakawane 
PR is ready for review . Please have a look and let me know if any changes required.

File: static/js/contacts.js [hunk 1]
```diff
@@ -54,12 +54,21 @@ const contactFormHTML = `
             </div>
             <div class="slack-container">
                 <div>
+                <div style="position: relative;">
                     <label for="slack-channel-id">Channel ID</label>
-                    <input type="text" class="form-control" id="slack-channel-id" required>
+                    <input type="text" class="form-control" id="slack-channel-id" style="position: relative;" required >
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+            title="Specify channel, private group, or IM channel (can be an encoded ID or a name) - required unless you provide a webhook."
+            id="info-slack-channel-id"></i>
```
Comment (nbhavana) [inline static/js/contacts.js:62]: remove the `- required unless you provide a webhook`

File: static/js/contacts.js [hunk 2]
```diff
@@ -54,12 +54,21 @@ const contactFormHTML = `
...
+            </div>
+                </div>
+                <div style="position: relative;">
+                <label for="slack-token">Slack API Token</label>
+                <input type="text" class="form-control" id="slack-token" style="position: relative;" required>
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+                    title="Provide a Slack API token (starts with “xoxb”) - required unless you provide a webhook."
+                    id="info-slack-token"></i>
```
Comment (nbhavana) [inline static/js/contacts.js:70]: remove the `- required unless you provide a webhook`

**Comment by PratikshaPatil77** [comment]
@nbhavana 
Changes are done.Please you can have a look.

File: static/js/contacts.js [hunk 3]
```diff
@@ -54,12 +54,21 @@ const contactFormHTML = `
...
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+            title="Specify channel, private group, or IM channel (can be an encoded ID or a name)."
+            id="info-slack-channel-id"></i>
+            </div>
+                </div>
+                <div style="position: relative;">
+                <label for="slack-token">Slack API Token</label>
+                <input type="text" class="form-control" id="slack-token" style="position: relative;" required>
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+                    title="Provide a Slack API token (starts with “xoxb”)."
```
Comment (AndrewHess) [inline static/js/contacts.js:69]: `xoxb` tokens are specifically bot tokens: https://api.slack.com/authentication/token-types
 Are we only allowing/expecting bot tokens? If so the tooltip should mention that

Comment (sonamgupta21) [reply static/js/contacts.js:69]: Yes, we only support bot tokens.

**Comment by sonamgupta21** [comment]
@PratikshaPatil77  can you please update the tooltip to 
`Provide a Slack bot API token (starts with "xoxb").`

**Comment by PratikshaPatil77** [comment]
Done.

**Review by nbhavana — APPROVED** [review]

**Pull request merged**